public/scripts/main.js

```diff
@@ -422,3 +423,24 @@ window.addEventListener('focus', function (e) {
     onSuccess();
   }
 });
+
+function createUserAnswer(questionNo, isCorrect, elapsedTimeMs){
+  let requestBody = {
+    userId: 'guest',
+    isCorrect: isCorrect,
+    elapsedTimeMs: elapsedTimeMs
+  }
+
+  fetch('http://localhost:3000/user/answers/question/' + questionNo, {
```

Comment:
Hey, hardcoding the address will be an issue.

Reply:
Oh! My mistake :flushed: . Fixed in the next commit